src/ui/public/inspector/ui/inspector_panel.js

```diff
@@ -47,7 +56,7 @@ class InspectorPanel extends Component {
 
   static getDerivedStateFromProps(nextProps, prevState) {
     if (hasAdaptersChanged(prevState.adapters, nextProps.adapters)) {
-      throw new Error('Adapters are not allowed to be changed on an open InspectorPanel.');
+      throw new Error(adapterChangedError);
```

Comment:
Also a developer facing exception in case of wrong API usage, so no need to translate.